pytest.ini

```diff
@@ -3,5 +3,6 @@ addopts = --cov . --pep8 --pylint --cov-report term --cov-report html --ds=micro
 norecursedirs = node_modules .git .tox static templates .* CVS _darcs {arch} *.egg
 pep8ignore =
    */migrations/* ALL
+   */financialaid/*.py W503
```

Comment:
W503 references line breaks before/after operators where the linter disagrees with PEP8 (https://github.com/PyCQA/pycodestyle/issues/197) and (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).  It's kind of a controversial topic IIRC, we're happy to go with whichever style you prefer.  Would you like us to remove this change and update lint errors instead?